windows/win_file.ps1

```diff
@@ -50,6 +50,33 @@ $result = New-Object psobject @{
     changed = $FALSE
 }
 
+# Used to delete directories and files with logic on handling symbolic links
+Function Delete($file)
+{
+    $filepath = $file.FullName
+    If ( $file.Attributes.ToString().contains("ReparsePoint") )
+    {
+        cmd.exe /c rmdir $filepath
```

Comment:
I don't love this either, but the P/Invoking the Win32 side would be a bit of pain (esp junction points via DeviceIoControl), fsutil has its own warts for this, and the PS FileSystem provider doesn't get symlink support until 5.0... So this works for now. Especially given the recursion, we should Try/Catch around this and include the filepath with the error message from cmd...